contributing/DOCS-CONTRIBUTING.md

```diff
@@ -68,45 +71,203 @@ documentation efforts, who are happy to help you get started.
-  did?
+- Look for a
+  [Good First Issue](https://github.com/knative/docs/labels/kind%2Fgood-first-issue)
+  in the backlog.
 
-### Submitting documentation PRs - what to expect
+- Try out Knative and send us feedback. For example, run through one of the
+  [install guides](../install/README.md) and then try
+  [Getting Started with Knative Serving](../install/getting-started-knative-app.md).
+
+    You should keep a
+    [friction log](https://devrel.net/developer-experience/an-introduction-to-friction-logging)
+    of your experience and then use that to open your first set of PRs.
+    Examples:
+
+   - What was hard for you?
+   - Did you stumble on something because it wasn't clear?
+   - Was a dependency not mentioned?
+
+**A few pointers and other considerations**
+
+  - Think about how you can improve the documentation as a whole. <br>
+    For example, how can you fix the issue you found so that others don't run
+    into the same challenges?
 
-Here's what will happen after you open a PR in the Docs repo:
+  - Are there multiple places that could be fixed?
 
-1. One of the Docs approvers will triage the PR and provide an initial
-   documentation review as soon as possible.
-   1. If the PR involves significant changes or additions to sample code, we'll
-      flag it for engineer review.
-1. Once we're satisfied with the quality of the writing and the accuracy of the
-   content, we'll approve the change.
+     - Are there other pages that you should apply your update?
+
+     - Would it help if there was a link to more or related information on a
+       the page? On a related page?
+
+  - Was the title or description misleading? Did you expect to find something
+    else?
+
+  - If you find something and dont have the bandwidth to open an actual PR, help
+    make us aware of the pain point and open an
+    [Issue](https://github.com/knative/docs/issues/new).
+
+### Submitting documentation PRs - what to expect
 
-If you're making a change to the documentation, you should submit a PR against
-the master branch.
+Here's what generally happens when you open a PR against the `knative/docs` repo:
+
+1. One of the assigned repo maintainers will triage the PR by assigning relative
+   priority, adding appropriate labels, and performing an initial documentation
+   review.
+1. If the PR involves significant technical changes, for example new features,
+   or new and changed sample code, the PR is assigned to a Subject Matter
+   Expert (SME) or engineer for technical review and approval.
+1. When both the technical writers and SMEs are satisfied with the quality
+   of the writing and the technical accuracy of the content, the PR can be
+   merged. A PR requires two labels before for it's eligible for merging:
+   `lgtm` and `approved`.
+   * The SME is responsible for reviewing the technical accuracy and is who
+     can provide the `lgtm`.
+
+   * The Knative technical writers generally use the publicly available
```

Comment:
Two thoughts here:

- The info about us using the style guide feels out of place in this area of the doc, which is focused on how to get the right labels for a PR to merge

- We should probably refer to "The Knative Docs repo approvers", and then link to https://github.com/knative/docs/blob/master/OWNERS_ALIASES#L9 so it's clear who we're referring to.

Reply:
i added this for a couple of reasons, i think it is in context of how to achieve the approved label, and we also dont have another doc where this info is covered. if we have new contributors reference this guide for how to get a PR merged, it should be thorough.